fips-pom.xml

```diff
@@ -2,7 +2,14 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>com.snowflake</groupId>
+
+    <parent>
+        <groupId>com.snowflake</groupId>
+        <artifactId>snowpark-parent</artifactId>
+        <version>1.10.0-SNAPSHOT</version>
+        <relativePath>./parent-pom.xml</relativePath>
+    </parent>
```

Comment:
Double check on whether our POM test checks for parents or not.

Reply:
It passed the merging gate